### PR TITLE
ADD logEI (according to Hutter et al. LION'11)

### PR DESCRIPTION
**Pull request by mlindauer**
* ADD logEI (according to Hutter et al. LION'11)
* RM derivative argument in acq methods
* MAINT some cleanup in acquisition.py

### CODE REVIEW

File: smac/smbo/acquisition.py [hunk 1]
```diff
@@ -208,12 +183,9 @@ def __init__(self,
...
         Parameters
         ----------
-        model : Model object
+        model : AbstractEPM
             A model that implements at least
-                 - predict(X)
-                 - getCurrentBestX().
-            If you want to calculate derivatives than it should also support
-                 - predictive_gradients(X)
+                 - predict_marginalized_over_instances(X)
```
Comment (mlindauer) [inline smac/smbo/acquisition.py]: I don't know the entire EIPS implementation, but from the code, I would guess that `predict_marginalized_over_instances(X)` returns both cost and running time.

Comment (KEggensperger) [inline smac/smbo/acquisition.py]: I would assume the same. @mfeurer added EIPS, so I'll ask him to add documentation.

**Comment by mlindauer** [comment]
How to proceed with this PR? It's open for more than one week.

**Comment by KEggensperger** [comment]
Just a reminder, in case we merge that: Also close #203

**Pull request merged**